src/applications/disability-benefits/526EZ/helpers.jsx

```diff
@@ -126,6 +126,7 @@ export const facilityDescription = ({ formData }) => {
 export const treatmentView = ({ formData }) => {
   const { from, to } = formData.treatmentDateRange;
 
+  const name = formData.treatmentCenterName.label || '';
```

Comment:
If no label is provided, is a value provided that can be used instead?

Reply:
I don't think so.

Reply:
Thanks for explaining, does this mean that if there is no label, it is because there is no treatmentCenterName to being with?

Reply:
Yeah, exactly.

Reply:
Thanks again, does this throw a "cannot read label property of undefined" if no treatmentCenterName is provided?